lib/server.js

```diff
@@ -124,8 +124,9 @@ class S3Server {
                 address: ipAddress || '[::]',
                 port,
             };
-            logger.info('server started', { address: addr.address,
-                port: addr.port, pid: process.pid });
+            const { address } = addr;
```

Comment:
not sure if I'm missing anything but shouldn't `port` be added in here as well? (in order to support being referred directly as `port` instead of `addr.port` on the below lines)
```suggestion
            const { address, port } = addr;
```

Reply:
nvm, just read again.